### PR TITLE
PEO-430 Update rails_best_practices runner

### PR DESCRIPTION
**Pull request by rostikkkk2**
[PEO-430](https://garage.atlassian.net/browse/PEO-430)

### Description

Need to provide next ability:
load a configuration file form the application if it exists
otherwise, load default configuration file (from the config/rails_best_practices/config.yml)

### Before submitting the merge request make sure the following are checked

- [ ] Followed the style guidelines of this project
- [ ] Performed a self-review of own code
- [ ] Wrote the tests that prove that fix is effective/that feature works

### CODE REVIEW

File: config/rails_best_practices/config.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+AddModelVirtualAttributeCheck: { }
+AlwaysAddDbIndexCheck: { }
+#CheckSaveReturnValueCheck: { }
```
Comment (chubchenko) [inline config/rails_best_practices/config.yml]: Maybe would be better to remove commented checks?

File: lib/inquisition/rails_best_practices/runner.rb [hunk 2]
```diff
@@ -3,14 +3,23 @@
...
       end
 
+      private
+
       def options
-        { 'config' => 'rails_best_practices.yml', 'silent' => true }
+        { 'config' => choose_config_path, 'silent' => true }
+      end
+
+      def choose_config_path
```
Comment (chubchenko) [inline lib/inquisition/rails_best_practices/runner.rb]: Would be better to rename `choose_config_path` to `config_path`

File: lib/inquisition/rails_best_practices/runner.rb [hunk 3]
```diff
@@ -3,14 +3,23 @@
 module Inquisition
   module RailsBestPractices
     class Runner < ::Inquisition::Runner
+      USER_CONFIG_PATH = 'rails_best_practices.yml'.freeze
+      CUSTOM_CONFIG_PATH = File.join(File.dirname(__FILE__), '../../../config/rails_best_practices/config.yml')
```
Comment (chubchenko) [inline lib/inquisition/rails_best_practices/runner.rb]: Use `Inquisition.root`

File: lib/inquisition/rails_best_practices/runner.rb [hunk 4]
```diff
@@ -3,14 +3,23 @@
 module Inquisition
   module RailsBestPractices
     class Runner < ::Inquisition::Runner
+      USER_CONFIG_PATH = 'rails_best_practices.yml'.freeze
```
Comment (chubchenko) [inline lib/inquisition/rails_best_practices/runner.rb]: From the readme of the gem `configuration file location (defaults to config/rails_best_practices.yml)`

File: lib/inquisition/rails_best_practices/runner.rb (hunk 3 above)
Comment (chubchenko) [inline lib/inquisition/rails_best_practices/runner.rb]: Strange constant name

File: lib/inquisition/rails_best_practices/runner.rb (hunk 4 above)
Comment (chubchenko) [inline lib/inquisition/rails_best_practices/runner.rb]: Strange constant name
Also, maybe we can use constant from gem?

**Review by chubchenko — CHANGES_REQUESTED** [review]
Tests are absent

File: lib/inquisition/rails_best_practices/runner.rb (hunk 4 above)
Comment (rostikkkk2) [reply lib/inquisition/rails_best_practices/runner.rb]: There is no constant in gem, they just write `File.join(..., 'config/rails_best_practices.yml')`

**Pull request merged**